src/legacy/core_plugins/kibana/public/visualize/legacy.ts

```diff
@@ -27,9 +27,7 @@ import { plugin } from './index';
   const instance = plugin({} as PluginInitializerContext);
   instance.setup(npSetup.core, {
     ...npSetup.plugins,
-    __LEGACY: {
-      legacyChrome,
-    },
+    npData: npSetup.plugins.data,
```

Comment:
I missunderstood @pgayvallet in my first comment, I thought the import was just moved, but it's in fact a new one. Please just call it `data` within the plugin, then 
```ts
instance.setup(npSetup.core, npSetup.plugins);
```

Is enough. I have a draft PR up to clean up the other shims in the same way

Reply:
Done.